node/executor/src/lib.rs

```diff
@@ -117,7 +117,7 @@ mod tests {
 	}
 
 	fn executor() -> ::substrate_executor::NativeExecutor<Executor> {
-		substrate_executor::NativeExecutor::new(None)
+		substrate_executor::NativeExecutor::new(Default::default(), None)
```

Comment:
I think this grown unwieldy at this point. Should we provide an associated function on `NativeExecutor` to create an instance configured with defaults?

Reply:
We could implement `Default` for `NativeExecutor`?

Reply:
Yeah you got me: I have a bit of allergy on it since it is being overused. You don't have to go far for an example, you can see it in the diff: `Default::default()`. Yes, it is easier to write and it doesn't need to be updated if the type of an argument changes, but I would argue that one should optimize for readability not convenience of writing.

I won't block merge a PR that implements `Default` for `NativeExecutor` though, I think at this moment it is just matter of my preference. So your call.

Reply:
Decided to remove the `impl Default for WasmExecutionMethod` and pass it explicitly everywhere.